src/DynamoUtilities/Md2Html.cs

```diff
@@ -93,53 +96,63 @@ protected static string GetToolPath(string relativePath)
             var toolPath = Path.Combine(rootPath,relativePath);
             return toolPath;
         }
-        //TODO if we see any issues with deadlocks we can try using a timeout on another thread.
         /// <summary>
         /// Read data from CLI tool
-        /// <returns>Returns data read from CLI tool</returns>
         /// </summary>
-        protected virtual string GetData()
+        /// <param name="timeoutms">will return empty string if we don't finish reading all data in the timeout provided in milliseconds.</param>
+        /// <returns></returns>
+        protected virtual async Task<string> GetData(int timeoutms)
```

Comment:
@sm6srw @pinzart90 I was considering pulling this from a configuration file in the unfortunate case that a user is on a very slow machine, they could have some control over the timeout to avoid never getting in depth documentation or feature flags...

Reply:
I guess a timeout could happen once in a while on heavy loaded machine. Using a machine where the timeout happens all the time must be a bad experience in general. I think the risk for a timeout is higher for the feature flags as both load and/or network can cause it. Maybe add the value to a file if it is easy? The question then would be one or two values?

Reply:
Let's test out the timeouts by blocking the thread on the featureflags exe side.
Maybe we should just have a dedicated config to turn off ff completely and keep the timeout to something sensible by default

Reply:
it's not just FF, but all of these CLIWrapper base tools (like md2html.exe)

`Let's test out the timeouts by blocking the thread on the featureflags exe side.`
I've already tested locally by immediately exiting from the process or somehow stopping std out from being written to - is that what you mean?

I just think it's hard to pick a sensible timeout for all user scenarios though I guess it's not the end of the world if feature flags report their default or the user never gets in depth documentation.

Reply:
discussed with @pinzart90 I've increased these and I think it's more sensible now.